crates/wiggle/src/lib.rs

```diff
@@ -5,7 +5,10 @@ use syn::parse_macro_input;
 
 #[proc_macro]
 pub fn from_witx(args: TokenStream) -> TokenStream {
-    let config = parse_macro_input!(args as wiggle_generate::Config);
+    let mut config = parse_macro_input!(args as wiggle_generate::Config);
+    config
+        .witx
+        .make_paths_relative_to(env!("CARGO_MANIFEST_DIR"));
```

Comment:
Is this intended to be `std::env::var`? Otherwise it looks like paths will be relative to the `wiggle` crate, and I'm not sure if that's what's intended?

Reply:
I think my push of that fix raced your review :)